src/libaktualizr/config/config.h

```diff
@@ -100,7 +100,8 @@ class Config : public BaseConfig {
   void updateFromPropertyTree(const boost::property_tree::ptree& pt) override;
   void updateFromCommandLine(const boost::program_options::variables_map& cmd);
 
-  std::vector<boost::filesystem::path> config_dirs_ = {"/usr/lib/sota/conf.d", "/var/sota/sota.toml", "/etc/sota/conf.d/"};
+  std::vector<boost::filesystem::path> config_dirs_ = {"/usr/lib/sota/conf.d", "/var/sota/sota.toml",
+                                                       "/etc/sota/conf.d/"};
```

Comment:
No actual changes here, right? As this might cause conflicts with a previous patch that adds /var/sota/sota.toml it would be better to drop from this patch.

Reply:
Dangit. 'ninja format' keeps doing that and then I do a commit -a. I'll fix

Reply:
Dangit. 'ninja format' keeps doing that and then I do a commit -a. I'll fix